vm-setup/roles/v1aX_integration_test/templates/controlplane_centos.yaml

```diff
@@ -60,8 +63,8 @@ spec:
     - chmod a+x $(pwd)/{kubeadm,kubelet,kubectl}
     - mv $(pwd)/{kubeadm,kubelet,kubectl} /usr/local/bin/
     - mkdir -p /etc/systemd/system/kubelet.service.d
-    - curl -sSL "https://raw.githubusercontent.com/kubernetes/kubernetes/{{KUBERNETES_BINARIES_VERSION}}/build/debs/kubelet.service" | sed "s:/usr/bin:/usr/local/bin:g" > /etc/systemd/system/kubelet.service
-    - curl -sSL "https://raw.githubusercontent.com/kubernetes/kubernetes/{{KUBERNETES_BINARIES_VERSION}}/build/debs/10-kubeadm.conf" | sed "s:/usr/bin:/usr/local/bin:g" > /etc/systemd/system/kubelet.service.d/10-kubeadm.conf
+    - curl -sSL "https://raw.githubusercontent.com/kubernetes/release/v0.2.7/cmd/kubepkg/templates/latest/deb/kubelet/lib/systemd/system/kubelet.service" | sed "s:/usr/bin:/usr/local/bin:g" > /etc/systemd/system/kubelet.service
```

Comment:
what is this version ?

Reply:
As of k8s version, the config's and binaries' versions are decoupled. 
The current config version is v.0.2.7 which is compatible with previous versions. Please see the following issue, which I have reported earlier. 
https://github.com/kubernetes/kubernetes/issues/89757
and the documentation.
https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#installing-kubeadm-kubelet-and-kubectl

Reply:
And, here is the fix that introduced the v.0.2.7. https://github.com/kubernetes/website/pull/20021